src/client/elm/Explorer/View.elm

```diff
@@ -1865,6 +1867,26 @@ viewEventUpdateEnueuedDetails ctx event =
         BakerStakeThresholdPayload threshold ->
             paragraph [] [ text <| "Update the minimum staked amount for becoming a baker to " ++ T.amountToString threshold ]
 
+        UpdateAddAnonymityRevokerPayload (ArInfo anonymityRevokerInfo) ->
+            paragraph [] <| text ("Add a new anonymity revoker. ") :: displayArIp anonymityRevokerInfo
+
+        UpdateAddIdentityProviderPayload (IpInfo identityProviderInfo) ->
+            paragraph [] <| text ("Add a new identity provider. ") :: displayArIp identityProviderInfo
+
+
+displayArIp : ArIpInfo -> List (Element Msg)
+displayArIp info =
+    let descr = info.description
+    in [ text <| "Name: " ++ descr.name
+                 ++ ". Identity: " ++ String.fromInt info.identity
+                 ++ ". Description: " ++ descr.description
```

Comment:
Have you checked how multiline descriptions are displayed?

Reply:
Yes, they are displayed fine--at least on Firefox (in fact, that's what I tried to address in the other PR, #6). Or did you mean that it might be nicer to display this with explicit line breaks?

Reply:
Okay, great 😊 No, just that it was displayed nicely.